DorisProject/src/environments/environment.prod.ts

```diff
@@ -1,3 +1,10 @@
 export const environment = {
-  production: true
+  production: true,
+  apiDoris: 'https://dorisprojectbackend-preprod.herokuapp.com',
+  getStock: '/api/getproductsCategory',
+  login: '/api/v1/userValidation',
+  saveProduct: '/api/saveProduct',
+  getProductSku: '/api/getProduct/',
+  getCategoryList: '/api/getCategoryList/',
+  saveCategory: '/api/category'
```

Comment:
Codacy has a fix for the issue: [Insert `,`](https://app.codacy.com/gh/SaraArenasV/DorisProjectFrontEnd/pullRequest?prid=9257715)
```suggestion
  saveCategory: '/api/category',
```